pages/en/supported-networks/cosmos.mdx

```diff
@@ -0,0 +1,246 @@
+
+## Querying a Cosmos subgraph
+
+The GraphQL endpoint for Cosmos subgraphs is determined by the schema definition, with the existing API interface. Please visit the [GraphQL API documentation](https://thegraph.com/docs/en/developer/graphql-api/) for more information.
+
+## Supported Cosmos Blockchains
+
+### Cosmos Hub
+
+##### What is Cosmos Hub?
+
+The [Cosmos Hub blockchain](https://hub.cosmos.network/) is the first blockchain in the [Cosmos](https://cosmos.network/) ecosystem. You can visit the [official documentation](https://docs.cosmos.network/) for more information.
+
+##### Subgraph Dependencies
+
+[graph-cli](https://github.com/graphprotocol/graph-cli) is a CLI tool to build and deploy subgraphs, version `>=0.30.0` is required in order to work with Cosmos Hub subgraphs.
+
+[graph-ts](https://github.com/graphprotocol/graph-ts) is a library of subgraph-specific types, version `>=0.27.0` is required in order to work with Cosmos Hub subgraphs.
+
+##### Networks
+
+Cosmos Hub mainnet is `cosmoshub-4`. Cosmos Hub current testnet is `theta-testnet-001`. <br/>Other Cosmos Hub networks, i.e. `cosmoshub-3`, are halted, therefore no data is provided for them.
+
+##### Example Subgraphs
+
+Here are some example subgraphs for reference:
+
+[Cosmos Hub Block Filtering Example](https://github.com/graphprotocol/example-subgraph/tree/cosmos-block-example)
+
+[Cosmos Hub Validator Rewards Example](https://github.com/graphprotocol/example-subgraph/tree/cosmos-event-example)
+
+[Cosmos Hub Validator Delegations Example](https://github.com/graphprotocol/example-subgraph/tree/cosmos-transaction-example)
+
+### Osmosis
+
+##### What is Osmosis?
+
+[Osmosis](https://osmosis.zone/) is a decentralized, cross-chain automated market maker (AMM) protocol build on top of the Cosmos SDK. It allows users to create custom liquidity pools and trade IBC enabled tokens. You can visit the [official documentation](https://docs.osmosis.zone/) for more information.
+
+##### Subgraph Dependencies
+
+[graph-cli](https://github.com/graphprotocol/graph-cli) is a CLI tool to build and deploy subgraphs, version `>=0.30.0` is required in order to work with Osmosis subgraphs.
+
+[graph-ts](https://github.com/graphprotocol/graph-ts) is a library of subgraph-specific types, version `>=0.27.0` is required in order to work with Osmosis subgraphs.
+
+##### Networks
+
+Osmosis mainnet is `osmosis-1`. Osmosis current testnet is `osmo-test-4`.
+
+##### Example Subgraphs
+
+Here are some example subgraphs for reference:
+
+[Osmosis Block Filtering Example](https://github.com/graphprotocol/example-subgraph/tree/osmosis-block-example)
+
+[Osmosis Validator Rewards Example](https://github.com/graphprotocol/example-subgraph/tree/osmosis-event-example)
+
+[Osmosis Validator Delegations Example](https://github.com/graphprotocol/example-subgraph/tree/osmosis-transaction-example)
```

Comment:
Don't we want a shared section for this? With the current structure we would link the Cosmos examples here (with templating).